scaling_performance/optimizing_storage.adoc

```diff
@@ -322,18 +322,50 @@ Red Hat strongly recommends using the `DeviceMapper` storage driver in thin-pool
 potentially improve density by reducing overall memory utilization.
 
 [[benefits-of-using-the-overlay-graph-driver]]
-== Benefits of Using the Overlay Graph Driver with SELinux
+== Comparing the overlay vs. overlay2 Graph Drivers 
 
-The default Docker storage configuration on Red Hat Enterprise Linux continues
-to be `DeviceMapper`. While the use of `Overlay` as the container's storage
-technology is under evaluation, moving Red Hat Enterprise Linux to `Overlay` as
-the default in future releases is under consideration. As of Red Hat Enterprise
-Linux 7.2, `Overlay` became a supported graph driver. As of Red Hat Enterprise
-Linux 7.4, SELinux and the `Overlay2` graph driver became a supported
+OverlayFS is a type of union file system. It allows the user to overlay one file system on top of another. 
+Changes are recorded in the upper file system, while the lower file system remains unmodified. 
+This allows multiple users to share a file-system image, such as a container or a DVD-ROM, where the base image is on read-only media. 
+
+OverlayFS layers two directories on a single Linux host and presents them as a single directory. These directories are called layers 
+and the unification process is referred to as a union mount. 
+
+OverlayFS uses one of two graph drivers, *overlay* or *overlay2*. As of Red Hat Enterprise
+Linux 7.2, *overlay* graph river became a supported graph driver. As of Red Hat Enterprise
```

Comment:
The following is saying on rhel7.4, selinux + overlay2 is supported, then here on rhel7.2, overlay + selinux is not supported, if yes, it is better to mention selinux + overlay is not supported on rhel7.2.